tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+## Purpose
+This lesson describes the steps for configuring a virtual machine to run Kite example code on a Cloudera Quickstart VM.
+
+### Result
+Your VM is ready for you to install and run sample programs from the Kite SDK Examples project.
+
+## Installing the VM and Kite Examples
+
+1. Install a VirtualBox or VMWare [Cloudera QuickStart VM][getvm] version 5.2 or later.
```

Comment:
Install is fine.

My original draft had the configuration for CM, but I was told to remove it. I can add it back in.